ted_sws/master_data_registry/services/rdf_fragment_processor.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+
+# rdf_fragment_processor.py
+# Date:  29.07.2022
+# Author: Stratulat Ștefan
+# Email: stefan.stratulat1997@gmail.com
+
+"""
+
+"""
+import pathlib
+from string import Template
+from typing import List
+
+from rdflib import Graph
+
+from ted_sws.master_data_registry.model.rdf_fragment import RDFFragment
+from ted_sws.master_data_registry.resources import RDF_FRAGMENT_BY_URI_SPARQL_QUERY_TEMPLATE_PATH, \
+    TRIPLES_BY_CET_URI_SPARQL_QUERY_TEMPLATE_PATH
+
+DEFAULT_RDF_FILE_FORMAT = "n3"
```

Comment:
why `n3` and not Turtle?

Reply:
n3 is used also for Turle, based on rdflib documentation